src/notebooks/notebookEditorProvider.ts

```diff
@@ -46,4 +51,52 @@ export class NotebookEditorProvider implements INotebookEditorProvider {
         const doc = await this.vscodeNotebook.openNotebookDocument(JupyterNotebookView, data);
         await this.vscodeNotebook.showNotebookDocument(doc);
     }
+
+    registerEmbedNotebookProvider(provider: IEmbedNotebookEditorProvider): void {
+        this.providers.add(provider);
+    }
+
+    findNotebookEditor(resource: Resource) {
+        const key = resource ? getComparisonKey(resource, true) : 'false';
```

Comment:
will any actually use the comparison key 'false'?

Reply:
or the next several lines just be skipped

Reply:
oh this was all just moved, probably just don't worry about it for now then

Reply:
I don't know, it was copied from the original implementation though